test_scripts/Polices/build_options/031_ATF_P_TC_PTU_Retry_Sequence_Retry_Timeout_Computation_HTTP.lua

```diff
@@ -136,20 +137,46 @@ function Test.DeletePTUFile()
 end
 
 function Test:ValidatePTS()
-  if ptu.policy_table.consumer_friendly_messages.messages then
-    self:FailTestCase("Expected absence of 'consumer_friendly_messages.messages' section in PTS")
+  if(ptu == nil) then
+    self:FailTestCase("Binary data is empty")
+  else
+    if ptu.policy_table.consumer_friendly_messages.messages then
+      self:FailTestCase("Expected absence of 'consumer_friendly_messages.messages' section in PTS")
+    end
   end
 end
 
-function Test.UpdatePTS()
-  ptu.policy_table.device_data = nil
-  ptu.policy_table.usage_and_error_counts = nil
-  ptu.policy_table.app_policies["0000001"] = { keep_context = false, steal_focus = false, priority = "NONE", default_hmi = "NONE" }
-  ptu.policy_table.app_policies["0000001"]["groups"] = { "Base-4", "Base-6" }
-  ptu.policy_table.functional_groupings["DataConsent-2"].rpcs = json.null
-  -- updating parameters in order to speed up the process
-  ptu.policy_table.module_config.seconds_between_retries = { 1, 1, 1 }
-  ptu.policy_table.module_config.timeout_after_x_seconds = 10
+function Test:UpdatePTS()
+  if(ptu == nil) then
+    local config_path = commonPreconditions:GetPathToSDL()
+    local pathToFile = config_path .. 'sdl_preloaded_pt.json'
+
+    local file = io.open(pathToFile, "r")
+    local json_data = file:read("*all")
+    file:close()
+
+    ptu = json.decode(json_data)
+
+    ptu.policy_table.device_data = nil
```

Comment:
@istoimenova, In order not to have duplicates of the same code (lines 160 - 167) it's better to define just ptu variable if it's nil and remain just only one copy of duplicated code